### ISSUE TITLE
Remarks to manual v4.3.0

### ISSUE THREAD

**Issue by RalfHerzog** [jenetics/jenetics#425]
Hi. I finished reading through your manual completely. It is written very well, but I found a few places which are unclear to me.

**Page 18**
For "Multi-point crossover" you show an example for the crossover at the indexes 1 and 4. Unfortunately I cannot see the use in the figure 1.3.10. Why are there 4 chromosome images instead of three (one for the initial chromosome and two for the recombination). How did they work?

**Page 25**
I think I found a typo at "This limited Engines no **long** creates [...]". Shouldn't it be "This limited Engines no **longer** creates [...]"?

**Page 29**
I think I found a typo at "[...] you have **alive** view on the evolution progress.". Do you mean "**a live**"?

**Page 88**
The "Pareto dominance" is defined as "[...] iff u is partially **less** than v, [...]" but in the following term (starting with "i.e.") only **greater** (or equal) than is used. Does the definition with "less" fit the term, since u_i must be **greater** than v_i for one or multiple "i"?

**Page 92**
Between the black lines you start with "Since the **MOO** classes [...]" but "MOO" (Multi-objective optimization) was never defined before, only MOP was.

**Comment by jenetics** [comment]
Thank you, I will have a look.

**Comment by jenetics** [comment]
**Page 18**
There is a slight mismatch between the description and the figures. What the figures shows is the original chromosomes selected for crossover and the crossover result for different indices. I clarified this, in the text and the images.

**Page 25**
You are right, has been fixed.

**Page 29**
Right, has been fixed.

**Page 88**
You are also right here. I took the definition as referenced and forgot to change this. In the literature Pareto dominance is defined with less, but the default for the optimization direction in Jenetics is greater. So I had to adapt the definition, to be consistent with the library. Otherwise the MOO would have a different optimization default.

**Page 92**
Also correct at this point. Has been clarified.


Thank you again for the careful reading of the manual!
